build.rs

```diff
@@ -287,6 +295,24 @@ fn read_env_var(name: &'static str) -> Result<String, std::env::VarError> {
     std::env::var(name)
 }
 
+// Find executable in PATH with optional extra search directories
+fn find_in_path(path: &Path, extra: Vec<&Path>) -> Option<PathBuf> {
+    #[cfg(target_os = "windows")]
+    let path = &path.with_extension("exe");
+
+    if let Some(found) = std::env::split_paths(&std::env::var_os("PATH").unwrap_or(OsString::new()))
```

Comment:
Do you think we could use [PathFindOnPath](https://docs.microsoft.com/en-us/windows/win32/api/shlwapi/nf-shlwapi-pathfindonpathw) Windows API here instead? 🤔

Reply:
That's a Windows API, and I already have dreams of fixing build.rs for cross-compilation from Linux to Windows one day :)

Reply:
Haha, makes sense 😅